client/me/purchases/billing-history/utils.tsx

```diff
@@ -229,6 +229,31 @@ function renderSpaceAddOnquantitySummary(
 	} );
 }
 
+export function renderTransactionVolumeSummary(
+	{ volume, wpcom_product_slug }: BillingTransactionItem,
+	translate: LocalizeProps[ 'translate' ]
+) {
+	if ( ! volume ) {
+		return null;
+	}
+
+	volume = parseInt( String( volume ) );
+
+	if ( 'dotblog_domain' !== wpcom_product_slug ) {
```

Comment:
I think we should include every domain registration products

Reply:
I agree here, not sure why we would limit it to just .blog domains if we have access to this information for all registrations 👍

Reply:
Thanks for catching this. Its been fixed.